export_mods_main.py

```diff
@@ -0,0 +1,95 @@
+"""Script that creates all directories"""
+import os
+
+# Change to the project repository location
+my_wd = os.getcwd()
+my_repo = "research-and-development"
+if not my_wd.endswith(my_repo):
+    os.chdir(my_repo)
+
+from src.utils.singleton_boto import SingletonBoto
+
+config = {
+    "s3": {
+        "ssl_file": "/etc/pki/tls/certs/ca-bundle.crt",
+        "s3_bucket": "onscdp-dev-data01-5320d6ca"
+    }
+}
+
+boto3_client = SingletonBoto.get_client(config)
+import src.utils.s3_mods as mods
```

Comment:
could we just move the boto3_client initialisation to export_files.py? (conditional on which platform we are using). We already have the s3 settings in the config. It would be nice if possible to get rid of this script before we merge.